sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -100,10 +99,11 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
       );
     }
 
-    this.azureRegion = options.regionalAuthority ?? process.env.AZURE_REGIONAL_AUTHORITY_NAME;
-    if (this.azureRegion === RegionalAuthority.AutoDiscoverRegion) {
-      this.azureRegion = "AUTO_DISCOVER";
-    }
+    // TODO: Uncomment once we're ready to release this feature.
+    // this.azureRegion = options.regionalAuthority ?? process.env.AZURE_REGIONAL_AUTHORITY_NAME;
```

Comment:
As I commented in the changelog, we intend to support the environment variable without explicitly documenting it to support first parties until the feature has been enabled for third parties as well.

Reply:
Ok ok thank you!